OneSignalSDK/onesignal/src/main/java/com/onesignal/BundleCompat.java

```diff
@@ -87,8 +87,11 @@ public void putBoolean(String key, Boolean value) {
    }
    
    @Override
-   public String getString(String key) {
-      return mBundle.getString(key);
+   public String getString(String key) throws OSBundleCompatException {
+      if (mBundle != null)
+         return mBundle.getString(key);
+      else
+         throw new OSBundleCompatException("unable to get mBundle " + this.toString());
```

Comment:
no toString method is implemented, is this printing all the data properly?

Reply:
You make a good point, since mBundle is the only piece of information valuable but is null at the moment, should I just remove it then?

Reply:
We already know there are some cases where the bundle is null. I think will give more information if places were processFromFCMIntentService is called catch the exception, in order to give more information about the cases where the bundle is null